docs/advanced.rst

```diff
@@ -656,6 +655,83 @@ To reduce boilerplate, you could create shortcuts, like so:
         name = json_parsed["name"]
         # ...
 
+Argument Passing and ``arg_name``
+---------------------------------
+
+.. NOTE::
+
+    This section describes behaviors which are planned to change in ``webargs``
+    version 9. In version 8, behavior will be as follows. In version 9,
+    ``USE_ARGS_POSITIONAL`` will be removed and will always be ``False``.
+
+By default, ``webargs`` provides two ways of passing arguments via decorators,
+`Parser.use_args <webargs.core.Parser.use_args>`, and `Parser.use_kwargs <webargs.core.Parser.use_kwargs>`.
+``use_args`` passes parsed arguments as positionals, and ``use_kwargs`` expands
+dict-like parsed arguments into keyword arguments.
+
+For ``use_args``, the result is that sometimes it is non-obvious which order
+arguments will be passed in. Consider the following two example snippets, one
+of which are nearly identical:
```

Comment:
> one of which are nearly identical

I'm not a native speaker but I find this sentence a bit strange. What about

> Consider the following nearly identical example snippets:

Reply:
Good catch! Thanks!

I think you're right to find it strange.
"one of which are" isn't grammatical as far as I know. I'm guessing that this was an editing error of mine and that my intent was to make it read
> Consider the following two example snippets, which are nearly identical:

I prefer the phrasing you proposed, "Consider the following nearly identical example snippets", so I'll go with that.
It's a more direct construction and, IMO, easier to read.